### PR TITLE
Minor docs fix

### PR DESCRIPTION
**Pull request by ecfm**
1. Make Config names in doc the same as labels displayed in UI
2. Change config name `Null Field Value` to `Ignore NULL Value`, and update its description

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,15 +5,13 @@
...
----------------------
+## Plugin Configuration
 
 | Config | Required | Default | Description |
 | :------------ | :------: | :----- | :---------- |
-| **Field** | **Y** | N/A | Input field that should be used for comparator. It has to be of type numeric (int, long, float and double).|
-| **Size** | **N** | 1 | Specifies the size of the top-N to be generated. If no of input records is less than N, then all records will ordered by the 'field' specified above.  |
-| **Null Field Value** | **N** | 'false' | Specifies the list of fields from the input that should be considered as hashing keys. All the fields should be non-null. Comma separated list of fields to be used as hash keys. |
+| **Top Field** | **Y** | N/A | Input field that should be used for comparator. It has to be of type numeric (int, long, float and double).|
+| **Results Number** | **N** | 1 | Specifies the size of the top-N to be generated. If no of input records is less than N, then all records will be ordered by the 'field' specified above.  |
```
Comment (deepakwadhwani) [inline README.md]: lets spell out number.

**Comment by awholegunch** [comment]
I've made numerous changes to the README. Please take a look, and copy the changes to the `TopN-batchaggregator.md` file.

File: README.md [hunk 2]
```diff
@@ -2,14 +2,14 @@
 
 # Top-N
 
-Top-N returns the top "n" records from the input set, based on a criteria specified in a field.
+Top-N returns the top "n" records from the input set, based on the criteria specified in plugin configuration.
```
Comment (awholegunch) [inline README.md]: the plugin configuration

**Comment by awholegunch** [comment]
One comment, otherwise README LGTM

**Pull request merged**